### PR TITLE
[DOCS] Update How to create Expectations Interactively in Python

### PR DESCRIPTION
**Pull request by kwcanuck**
## Description
A community member identified an error in step 3 of _[How to create Expectations Interactively in Python](https://docs.greatexpectations.io/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data/)_. The Slack conversation is located [here](https://greatexpectationslabs.slack.com/archives/C03B8DZCJ07/p1685658181605369).  This PR adds the requested edits.

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] Appropriate tests and docs have been updated

### CODE REVIEW

File: docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md [hunk 1]
```diff
@@ -96,20 +96,34 @@ You can provide a dictionary as the `options` parameter of `build_batch_request(
...
+
+    ```python 
+    context.create_expectation_suite("my_expectation_suite")
+    ```
+
+2. Optional. Run the following command if you haven't added or updated your Expectation Suite:
+
+    ```python 
+    context.add_or_update_expectation_suite("my_expectation_suite")
+    ```
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md]: These python code blocks need to be pulled from an example script that runs the code.

Comment (Rachel-Reverie) [reply docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md]: Additionally, only one of those lines is necessary.  I would stick with the first one: `create_expectation_suite`.  The error being addressed is cases when the Expectation Suite does not already exist, and for the purposes of creating it `create_expectation_suite` would be the more intuitive choice.

Comment (kwcanuck) [reply docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md]: @Rachel-Reverie I have removed the additional optional command as you recommended and added the code block.

File: tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py [hunk 2]
```diff
@@ -16,6 +16,11 @@
 batch_request = data_asset.build_batch_request()
 # </snippet>
 
+# <snippet name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py create_expectation_suite">
+context.create_expectation_suite("my_expectation_suite")
+assert "my_expectation_suite" in context.list_expectation_suite_names()
```
Comment (Rachel-Reverie) [inline tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py]: I would move this assert statement outside of the snippet.  We don't need to display it in the docs: it's just there to ensure that the code works as intended when we are testing the snippet.

**Review by Rachel-Reverie — APPROVED** [review]
LGTM, one minor suggestion for the change to the example script.

File: tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py (hunk 2 above)
Comment (kwcanuck) [reply tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py]: @Rachel-Reverie I've implemented the edit as requested.

**Pull request merged**